### PR TITLE
Added new landRegisterExtract attachment type

### PR DESCRIPTION
**Pull request by jensstalder**
## Semantic versioning

This PR is a **Minor** (non-breaking change, adds functionality or feature)  

## Generall things to consider 
- I have read the [**CONTRIBUTING.md**](https://github.com/qualipool/swissrets/blob/master/CONTRIBUTING.md) document.
- If my change requires documentation, I added it already.
- I have added tests to cover all my changes.
- All new and existing tests pass.

### Added
- New attachment element (landRegisterExtract) german: "Grundbuchauszug"

### CODE REVIEW

**Review by machss — APPROVED** [review]
LG2M

File: docs/Attachments.md [hunk 1]
```diff
@@ -14,3 +14,4 @@ linkVirtualTour | Virtual tour link, typically used for embeddings and integrati
 linkYoutube | Video link, typically used for embeddings and integrations.
 logo | Related logo image.
 plan | Floorplan image or document.
+landRegisterExtract | A document given to political officials (commune) with basic information portraying the property and it's registrational validity. (>= v2.2.0)
```
Comment (koffeinfrei) [inline docs/Attachments.md:17]: `it's` -> `its`

File: examples/full.xml [hunk 2]
```diff
@@ -206,6 +206,11 @@
               <title>The plan</title>
               <mimeType>image/jpeg</mimeType>
             </plan>
+            <landRegisterExtract>
+              <url>http://www.someorg.com/extract-file.pdf</url>
```
Comment (koffeinfrei) [inline examples/full.xml:210]: Minor: use `example.com`

File: examples/full.xml [hunk 3]
```diff
@@ -206,6 +206,11 @@
...
+              <title>Grundbuchauszug</title>
+              <mimeType>applicationn/pdf</mimeType>
```
Comment (koffeinfrei) [inline examples/full.xml:212]: `applicationn` -> `application`

Comment (machss) [reply examples/full.xml:212]: ..while on it: the existing schema contains two typos, too:
`Localized document file attachemnt` --> `attachment`

Comment (machss) [reply examples/full.xml:212]: ..and `An extra whitch causes additional costs.` --> `which`

**Pull request merged**